markdown/ch3.markdown

```diff
@@ -8,7 +8,7 @@
 > — William Blake, *The Tyger*
 
 You probably know that digital images are made up of *pixels*. These are
-the light signal *sampled on a regular grid*. When computing
+*samples* of the light signal *on a regular grid*. When computing
```

Comment:
"samples of a light signal on a grid" makes less sense than "sampling the light signal on a grid"; either revert or let's discuss.